File: best-practices/sdk-examples/typescript/basic-example.ts
import Anthropic from '@anthropic-ai/sdk';
import * as dotenv from 'dotenv';

// Load environment variables
dotenv.config();

async function main() {
    // Initialize the client
    const anthropic = new Anthropic({
        apiKey: process.env.ANTHROPIC_API_KEY,
    });
    
    // Create a message
    const message = await anthropic.messages.create({
        model: 'claude-3-5-sonnet-20241022',
        max_tokens: 1024,
        messages: [{
            role: 'user',
            content: 'Tell me a short joke about programming'
        }],
    });
    
    console.log("Claude's response:");
    console.log(message.content[0].text);
}

// Run the example
main().catch(console.error);

File: best-practices/sdk-examples/python/basic_example.py
#\!/usr/bin/env python3
"""
Basic Claude SDK Example
This script demonstrates simple message creation with the Claude API
"""
import os
from anthropic import Anthropic

def main():
    # Initialize the client
    client = Anthropic(
        api_key=os.environ.get("ANTHROPIC_API_KEY")
    )
    
    # Create a message
    message = client.messages.create(
        model="claude-3-5-sonnet-20241022",
        max_tokens=1024,
        messages=[
            {
                "role": "user",
                "content": "Tell me a short joke about programming"
            }
        ]
    )
    
    print("Claude's response:")
    print(message.content[0].text)

if __name__ == "__main__":
    main()
EOF < /dev/null